OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDiscussionResponsesFragment.java

```diff
@@ -133,14 +156,18 @@ public void onClick(View v) {
                 });
 
         addResponseLayout.setEnabled(!courseData.isDiscussionBlackedOut());
-    }
-
-    @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        EventBus.getDefault().register(this);
+        switch (discussionThread.getType()) {
```

Comment:
This switch-case should be refactor-moved to a new function `setScreenTitle()` and should be called before calling `loadThreadResponses()`;